pycalphad/codegen/callables.py

```diff
@@ -67,19 +71,53 @@ def build_callables(dbf, comps, phases, model=None, parameters=None, callables=N
         param_values = np.empty(0)
     comps = sorted(unpack_components(dbf, comps))
     pure_elements = get_pure_elements(dbf, comps)
+    # TODO: Conditions needing Hessians should probably have a 'second-order' tag or something
```

Comment:
It would make the code a little cleaner, at the cost of complexity. Right now the only 'second-order' condition is MU, so it would be unnecessary. I think it's okay to defer it to future conditions, such as when we explicitly support activity (AC) conditions.

Reply:
A complicating factor is sometimes we are just looking for the string 'MU', so the additional metadata gets lost. I'd prefer to defer it because it gets into a larger design issue about how to attach metadata to `StateVariable`s.